src/components/icons/mstable-icon.svg

```diff
@@ -0,0 +1,6 @@
+<svg viewBox="0 0 60 60" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Probably a dupe icon?

Reply:
Or do we remove the old one?

Reply:
Actually not! We were clipping the long logo before – will swap to new one though :)